src/routes/routes.js

```diff
@@ -15,6 +17,8 @@ import ProductsTable from '../pages/products-table';
 import ProductDetails from '../pages/product-details';
 import AboutUs from '../pages/about-us';
 
+import 'react-toastify/dist/ReactToastify.css';
```

Comment:
I think It's better to set this styles in `src/index.js`

Reply:
done